Makefile.am

```diff
@@ -161,4 +161,4 @@ pkgconfig_DATA= autotrace.pc
 ACLOCAL_AMFLAGS = -I m4
 
 check-recursive:
-	./tests/runtests.sh
+	AUTOTRACE="$(abs_builddir)/$(bin_PROGRAMS)" $(srcdir)/tests/runtests.sh
```

Comment:
The use of `$(bin_PROGRAMS)` feels a little icky here since what I really want is just the name of the autotrace program, not all programs. But since there is only one program right now, it works.